go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armsubscriptions v1.3.0
 	github.com/Masterminds/semver v1.5.0
 	github.com/Masterminds/sprig v2.22.0+incompatible
-	github.com/aws/aws-sdk-go v1.55.0
+	github.com/aws/aws-sdk-go v1.55.5
```

Comment:
I don't think this is necessary for the changes we are making

Reply:
right, but let's merge this. `make release-snapshot` sometimes fails (at least on mac) if we don't have this change.

Reply:
I removed this change.